clients/src/main/java/org/apache/kafka/common/security/ssl/SslFactory.java

```diff
@@ -95,6 +95,7 @@ public void configure(Map<String, ?> configs) throws KafkaException {
             try {
                 SslEngineValidator.validate(builder, builder);
             } catch (Exception e) {
+                log.error("SSLFactory configure failed during validate", e);
                 throw new ConfigException("A client SSLEngine created with the provided settings " +
```

Comment:
We should probably change the exception class here since `ConfigException` takes a config name and value while this one is passing in the exception message and cause like other exception constructors. Then existing log entry would be sufficient as well.